CHANGES_3.0.0.md

```diff
@@ -177,7 +173,14 @@ in on the options object . Additionally, `find` does not support individual opti
 `limit` as positional parameters. You must either pass in these parameters in the `options` object,
 or add them via `Cursor` methods like `Cursor.prototype.skip`.
 
-### Aggregation
+### `aggregate`
+
+`Collection.prototype.aggregate` no longer accepts variadic arguments. Pipeline stages
+are now only accepted as an `Array` of stages as the first argument.
```

Comment:
I think it would be helpful to show before and after images of the input shape here, make it very easy for people to see the difference. Also I think it might pay to explain our rationale for making the change (maintainability, consistency of api, etc)

Reply:
probably worth noting that things like this might have been included for shell compatibility